internal/action/command.go

```diff
@@ -437,9 +437,7 @@ func (h *BufPane) openHelp(page string, hsplit bool) error {
 		helpBuffer.SetOptionNative("hltaberrors", false)
 		helpBuffer.SetOptionNative("hltrailingws", false)
 
-		if h.Buf.Type == buffer.BTHelp {
-			h.OpenBuffer(helpBuffer)
```

Comment:
Is it a good idea to just remove this unconditionally? Shouldn't we add e.g. another flag for that (e.g. `-n` meaning "new pane"), or e.g. another option?

Reply:
When you use `> help` the first time it opens a new `hsplit`.
As user I would expect more or less the same when I perform the command a second time and thus I removed it.

But since someone could have been used to exactly that behavior the usual discussion could start again. So yes, I shouldn't remove it for that particular scenario. Maybe let it depend on the multi-open scenario?